libraries/AP_BattMonitor/AP_BattMonitor_UAVCAN.cpp

```diff
@@ -21,6 +21,17 @@
 
 extern const AP_HAL::HAL& hal;
 
+const AP_Param::GroupInfo AP_BattMonitor_UAVCAN::var_info[] = {
+
+    // @Param: CUR_MULT
+    // @DisplayName: Scales reported power monitor current
```

Comment:
Sorry I didn't notice this but ideally we should start the Display name with the subsystem.  So it should start with "Battery" or something like that.

Reply:
We also probably should have named this parameter "CURR_MULT" (two Rs) to be consistent with BATT_CURR_PIN.

Reply:
The subsytem BATT*_ will be prepended as it is part of the BATT*  table. PR for the name switch here https://github.com/ArduPilot/ardupilot/pull/19872 if you want to approve it ;)